llvm/test/Transforms/SROA/invariant-group.ll

```diff
@@ -138,10 +138,13 @@ end:
   ret void
 }
 
+; Conservatively do not split the alloca. 
 define void @partial_promotion_of_alloca() {
 ; CHECK-LABEL: @partial_promotion_of_alloca(
 ; CHECK-NEXT:    [[STRUCT_PTR_SROA_2:%.*]] = alloca i32, align 4
+; CHECK-NEXT:    [[STRUCT_PTR:%.*]] = alloca [[T:%.*]], align 4
```

Comment:
Uhm, I'm not entirely sure if it's fine in this specific case to have a field promoted to a scalar, and have the alloca still there.